src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,50 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+    private static final ManufacturerService manufacturerService
+            = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    private static final CarService carService
+            = (CarService) injector.getInstance(CarService.class);
+
     public static void main(String[] args) {
-        // test your code here
+        Manufacturer mersedes = new Manufacturer("Mersedes", "German");
+        Manufacturer toyota = new Manufacturer("Toyota", "Japan");
+        manufacturerService.create(mersedes);
+        manufacturerService.create(toyota);
+        manufacturerService.getAll().forEach(System.out::println);
+
+        Driver firstDriver = new Driver("Sam", "466574AS");
+        driverService.create(firstDriver);
+        Driver secondDriver = new Driver("Jon", "87dd788");
+        driverService.create(secondDriver);
+        driverService.getAll().forEach(System.out::println);
+
+        Car car = new Car("GLE2hCarDao011", mersedes, List.of(firstDriver, secondDriver));
+        Car car2 = new Car("RX222", toyota, List.of(firstDriver));
```

Comment:
List.of() will create immutable lists, and because of that you would need to make some workarounds in order for addDriverToCar() and removeDriverFromCar() mehtods to work. So consider using new ArrayList<>() here.